app/controllers/spree/admin/gdpr_requests_controller.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Spree
+  module Admin
+    class GdprRequestsController < ResourceController
+      before_action :load_data, only: :new
+
+      def index
+        respond_with(@collection, &:html)
```

Comment:
This reads a bit weird, because you usually see the longer syntax in Rails apps. I think I'd stick to the standards here.

Reply:
Rubocop told me to use this type of syntax instead of the longer one